pkgs/sdk/server-ai/Makefile

```diff
@@ -0,0 +1,8 @@
+build:
```

Comment:
TODO: I copied over this makefile from server, but these don't actually work because the csproj is in the src folder.